tests/in/functions-webgl.param.ron

```diff
@@ -1,6 +1,6 @@
 (
 	glsl: (
-		version: Embedded(300),
+		version: Embedded(310),
```

Comment:
This doesn't feel right. Test name is `functions-webgl`, but output glsl version is 3.1.